history-of-quakers/modernized/11-chapter-9.adoc

```diff
@@ -1699,14 +1760,23 @@ that all people may know, who hear it, that we came not in our own wills,
 but in the will of God.
 Given forth by me, who am known to men by the name of
 
-MARMADUKE STEVENSON.
+[.signed-section-signature]
+Marmaduke Stevenson.
 
 But having a new name given me, which the world knows not of, written in the Book of Life.
```

Comment:
Not sure if this should be labeled anything? It looks a little funny in the PDF in between the [sss] and the [sscc].